### DISCUSSION TITLE
纯小白，想知道假如添加cookie或refreshtoken加在哪里

### DISCUSSION THREAD

**Discussion by PoolarBBear** [DIYgod/RSSHub#12607] [Q&A] [answered]
本人白痴。用docker-compose.yml部署，现在人懵了，翻了文档半天也没看见，想知道cookie这些到底加在哪里，是environment吗？如果是的话等号后面用不用加双引号，麻烦发下源码看看谢谢

**Comment by TonyRL** [comment, marked as answer]
https://docs.rsshub.app
